lib/rqrcode/export/css.rb

```diff
@@ -0,0 +1,97 @@
+module RQRCode
+  module Export
+    module CSS
+
+      def as_css(options={})
+        id = options.fetch(:id, "qrcode")
+        clazz = options.fetch(:id, "qrcode-container")
+        base_size = options.fetch(:size, 200)
+        @@qr_logo = options.fetch(:logo, nil)
```

Comment:
Why do we need to class variables here..?
It is [bad style](https://github.com/rubocop-hq/ruby-style-guide#no-class-vars), and and it looks like it’s not threadsafe.

Reply:
how would you like to change it?

Reply:
I think we can just use instance variables: `@qr_logo` instead of `@@qr_logo`